lib/tbd/psi.rb

```diff
@@ -1840,9 +1884,10 @@ def process(model = nil, argh = {})
     edges.each do |identifier, edge|
       next unless  edge.key?(:psi)
       rsi        = 0
-      max        = edge[:psi].values.max
-      type       = edge[:psi].key(max)
+      max        = edge[:psi   ].values.max
+      type       = edge[:psi   ].key(max)
       length     = edge[:length]
+      length    *= edge[:mult  ] if edge.key?(:mult)
```

Comment:
TBD maintains the original OpenStudio vertices of subsurfaces with multipliers, yet adjusts the lengths of each  perimeter edge (i.e. of type "fenestration", or "head", "sill" or "jamb"). A TBD user can easily compare the JSON-reported edge vertices vs edge length to determine whether multipliers were factored in.

Reply:
`length` here is just a local variable for calculations right?  It doesn't get stored back into the TBD JSON model right?  You might call it `effective_length` or something to be more clear that it isn't physical length?

Reply:
Correct, local method variable. Doesn't feed back to JSON.